config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -5,7 +5,7 @@ periodics:
   agent: kubernetes
   decorate: true
   extra_refs:
-  - org: jetstack
```

Comment:
would it make sense to move all these test jobs into `config/jobs/cert-manager/cert-manager/...`? Then we have `config/jobs/{org}/{repo}` and we're consistent everywhere :)

Reply:
Done.